src/Rings/mpoly-ideals.jl

```diff
@@ -569,7 +572,7 @@ julia> minpoly(a)
 x^2 + 1
 ```
 """
-@attr function absolute_primary_decomposition(I::MPolyIdeal{<:MPolyRingElem{QQFieldElem}})
+@attr function absolute_primary_decomposition(I::T) where {T <: MPolyIdeal{<:MPolyRingElem{QQFieldElem}}}
```

Comment:
`T` not used yet.

Reply:
Removed the `T`. But it is not type stable anyways.